tests/items/getItems.test.ts

```diff
@@ -90,7 +90,12 @@ describe('GET /items', () => {
   });
 
   describe('should return 200 with an array of items with the "discount-switch-ssbu" item because user has a cart containing it, but it was not paid nor it is pending', () => {
-    for (const transactionState of [TransactionState.canceled, TransactionState.refunded, TransactionState.refused]) {
+    for (const transactionState of [
+      TransactionState.canceled,
+      TransactionState.refunded,
+      TransactionState.refused,
+      TransactionState.stale,
+    ]) {
       it(`should return 200 with array of items with the "discount-switch-ssbu" item because user has a cart containing it, but it was ${transactionState}`, async () => {
```

Comment:
"it was stale"
Noice. Je veux une ternaire (siteuplé)

Reply:
?

Reply:
dans le texte du it juste en dessous, ça donne
> because user has a cart containing it, but it was stale

Reply:
Toujours pas compris

Reply:
dans le for tu as un it. et vers la fin de la description du it tu as `because user has a cart containing it, but it was {transactionState}`
sauf qu'avec stale ça fait une phrase moch (bouboubouuuu)

Reply:
Ah si en vrai osef nan ...

Reply:
Mettre une ternaire ca va alourdir le code alors qu'on est sur un test

Reply:
hum... ;( jsuis triste. mais ok